kyo-prelude/shared/src/main/scala/kyo/Parse.scala

```diff
@@ -66,23 +66,27 @@ object Parse:
     private def fail(states: Seq[State], message: String)(using frame: Frame): Nothing < Abort[ParseFailed] =
         Abort.fail(ParseFailed(frame, states, message))
 
-    /** Tries all parsers in sequence and returns results from all successful parses. Unlike firstOf, this will evaluate all parsers even
```

Comment:
This scaladoc was incorrect. I think I changed the behavior of the method and forgot to update it.